src/App.tsx

```diff
@@ -61,7 +60,6 @@ const ROUTES_AUTHENTICATED = [
 
 export const App = () => {
   const { authorization, setAuthorization } = useAuthorization()
-  const apiUrl = useApiUrl()
   const navigate = useNavigate()
   const [loading, setLoading] = useState(true)
```

Comment:
There is a great lib names React Query. It helps handle state, errors and loadings

Reply:
Will be considered in the future. For now it's out of scope of this PR.